data/ca/legislature/Timothy-S-Grayson-6e51c9f1-a687-4907-91fb-92d48d6a2fbd.yml

```diff
@@ -0,0 +1,26 @@
+id: ocd-person/6e51c9f1-a687-4907-91fb-92d48d6a2fbd
+name: Timothy S. Grayson
+image: https://www.assembly.ca.gov/sites/assembly.ca.gov/files/memberphotos/ad14_header.jpg
+party:
+- name: Democratic
+roles:
+- start_date: '2018-12-03'
```

Comment:
@bfossen-ce from 12/05/2016 until 12/04/2022 this person represented district 14. on 12/05/2022 they began to rep district 15, so I'm not sure how this should be reflected/what their start date should be